content/en/docs/reference/tools/_index.md

```diff
@@ -39,7 +39,7 @@ Use Helm to:
 
 ## Kompose
 
-[`Kompose`](https://github.com/kubernetes/kompose) is a tool to help Docker Compose users move to Kubernetes.
+[`Kompose`](https://github.com/kubernetes/kompose) is a built-in tool to help Docker Compose users move to Kubernetes.
```

Comment:
We may want to avoid label a tool as "built-in". It carries more political bias than technical merits.

Reply:
will remove.

Reply:
removed.